addons/website_sale/static/tests/builder/products_item_option.test.js

```diff
@@ -0,0 +1,61 @@
+import { expect, test } from "@odoo/hoot";
+import { contains, defineModels, models } from "@web/../tests/web_test_helpers";
+import { patch } from "@web/core/utils/patch";
+import {
+    defineWebsiteModels,
+    setupWebsiteBuilder,
+} from "@website/../tests/builder/website_helpers";
+import { ProductsItemOptionPlugin } from "../../src/website_builder/products_item_option_plugin";
+
+class productRibbon extends models.Model {
+    _name = "product.ribbon";
+}
+
+defineWebsiteModels();
+defineModels([productRibbon]);
+
+patch(ProductsItemOptionPlugin.prototype, {
+    _saveRibbons() {
+        return true;
+    },
+});
```

Comment:
a better solution, it s to mock the rpc (with onRPC).
At the moment, your test doesn't really test anything. It's super whitebox.